zeek/README.md

```diff
@@ -0,0 +1,195 @@
+## Overview
+
+[Zeek][6] is the world's leading platform for network security monitoring. It interprets what it sees and creates compact, high-fidelity transaction logs, and file content. It can create fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
+
+This integration provides visualization and log enrichments for Connection Logs, DNS & DHCP Logs, Network Protocols, Files, Detections, and Miscellaneous event types.
+
+## Setup
+
+### Installation
+
+To install the Zeek integration, use the `datadog-agent integration install` command and follow the steps below. For more information, see the [Integration Management][7] documentation.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-zeek==1.0.0
+  ```
```

Comment:
The integration will be shipped by default with Agent 7.52+. This step is only needed for older Agent. Could you update that part? See https://github.com/DataDog/integrations-core/tree/master/tenable, which is also a log integration whipped with the agent